### PR TITLE
Update Unused Voicelines plugin to version 1.1

### PR DESCRIPTION
**Pull request by xNanochip**
The time when the game is over and getting ready to switch map, the scoreboard shows for way too long, making the server feel dormant. The match over music even finishes yet players are still waiting at the scoreboard screen waiting for the map to change for several seconds in silence. A small QoL fix to reduce this time to, at most, 10 seconds.

### CODE REVIEW

**Comment by xNanochip** [comment]
All of the changes in this update are to better replicate the natural announcer voicelines, class/character speeches, and the pre-game music that are in TF2's casual mode. None of these changes are invasive, distracting, nor denote from the normal tf2 casual experience, though this update brings cvars so that you can disable any sounds that you wish.

Also, I noticed the "Dont Fail Again" voiceline was commented out in the current version, therefore in this update I disabled this specific sound by default.

The source of this plugin can be found on my GitLab: https://gitlab.com/nanochip/unused-voicelines

Changes:
- added cvars
- added the casual mode pre-game startup music clip during the WaitingForPlayers period (pre-game)
- added attack/defend announcer sounds for both teams
- added payloadSetupFinished announcer sounds for both teams
- added class/character round start speeches
- reogranized and cleaned up a lot of code

File: roles/srcds/templates/server.cfg.j2 [hunk 1]
```diff
@@ -74,7 +74,7 @@ mp_forcecamera 1
 mp_enableroundwaittime 1
 mp_restartround 0
 mp_bonusroundtime 19
-mp_chattime 20
+mp_chattime 10
```
Comment (jack-avery) [inline roles/srcds/templates/server.cfg.j2:77]: `mp_chattime` is as high as it is right now to allow possible runoff mapvotes to pass before the mapchange occurs. If we lower this we'd need to tweak other things e.g. voting time, bonus round time, whether runoffs can occur etc... which I think as it is now is OK.

**Comment by jack-avery** [comment]
Tested on `cp_altitude`, `pl_upward`, `cp_granary`, and `cp_dustbowl`.

The sounds played by `uv_dontfailagain` play for the winning team on `cp_dustbowl`. I assume this will also be the case for `pl_thundermountain` as it's also multistage and the plugin doesn't account for it. It might be possible to remedy this using `full_round` through the `teamplay_round_win` hook (only completing the last stage of a map counts as a "full round").

File: roles/srcds/templates/server.cfg.j2 (hunk 1 above)
Comment (xNanochip) [reply roles/srcds/templates/server.cfg.j2:77]: Ah, race conditions at its finest, lol. That's okay though, maybe I'll have a look into the mapchooser and submit a different pr for this.

**Comment by xNanochip** [comment]
> Tested on `cp_altitude`, `pl_upward`, `cp_granary`, and `cp_dustbowl`.
> 
> The sounds played by `uv_dontfailagain` play for the winning team on `cp_dustbowl`. I assume this will also be the case for `pl_thundermountain` as it's also multistage and the plugin doesn't account for it. It might be possible to remedy this using `round_complete` of `teamplay_win_panel` which you've already hooked.

Good catch. The logic behind `uv_dontfailagain` should probably be changed to only play for the specific players who actually failed last round. As it currently stands, new players who join mid-match will also hear the sound even though they didn't contribute to the previous round's failure.

I've disabled this sound by default for the time being while I look into it further.

**Pull request merged**